algorithm/cgal/cgal.cc

```diff
@@ -5243,6 +5290,32 @@ int Outline(Geometry* geometry) {
   return STATUS_OK;
 }
 
+int Seam(Geometry* geometry, size_t count) {
```

Comment:
Is Seam already working? I thought the plan was to add that at some point in the future?

Reply:
Yeah, I wanted to see how it worked -- I think it'll require a bit of refinement, but it was easy enough to do.